python/CHANGELOG.rst

```diff
@@ -1,9 +1,12 @@
 --------------------
-[0.5.4] - 2023-01-13
+[0.5.4] - 2023-01-18
```

Comment:
0.5.4 is already released. Can you change the date back, and then make a new section at the top with the header:

```
--------------------
[0.5.5] - 2023-01-XX
--------------------
```
And place you changelog entry just below it?

Reply:
Yeah I wasn't certain what to do there. Didn't feel authority to start a new section!

Reply:
And the XX—leave as a variable, or use today's date?

Reply:
Ah. I'm seeing XX for other PRs